src/dato/lib/server.clj

```diff
@@ -116,13 +113,31 @@
         (casync/alt!
           tx-ch ([tx-report]
                  (println "TX-Broadcast Datoms:")
-                 (broadcast-tx! dato-server @all-sessions tx-report)
+                 (broadcast-tx! tal-state session-store tx-report)
                  (recur))
           stop-ch ([]
                    (println "Exiting tx-broadcast go-loop")
                    (casync/untap tx-mult tx-ch)))))
     stop-ch))
 
+(defn handle-websocket-msg-dispatch [tal-state dato-config session-store msg] (:op msg))
+(defmulti handle-websocket-msg #'handle-websocket-msg-dispatch)
```

Comment:
Is this a var during your dev, or is it redefined enough that it should be a var in production?